service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOController.java

```diff
@@ -47,6 +68,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
 
```

Comment:
Line above which I cannot comment on in this one:
        CaseManagementOrder caseManagementOrder = draftCMOService.getCaseManagementOrder(caseDetails);

What do you think about getCMO name? Would it be beneficial to refactor?

Reply:
I guess it's easier to scan read isn't it! Happy to change the method but I'd keep the caseData object non abbreviated.